pkg/controllers/management/clusterprovisioner/provisioner.go

```diff
@@ -966,10 +966,10 @@ func k3sClusterConfig(cluster *v3.Cluster) error {
 			Err:    fmt.Errorf("waiting for full cluster configuration"),
 			Reason: "Pending"}
 	}
-	// TODO Rancher may support upgrading the local cluster if its k3s in the future
-	if cluster.Name == v3.ClusterDriverLocal || cluster.Status.Driver == v3.ClusterDriverK3s {
-		return nil
+	if cluster.Status.Driver == v3.ClusterDriverK3s {
```

Comment:
I am not sure I understand why did you remove the exception for local clusters here

Reply:
@cjellick Asked that we be able to recognize local clusters if they are k3s. Do you see any issues here?